build-infra/build-vm.sls

```diff
@@ -206,29 +206,22 @@ github.com:
 {% endif %}
 
 {% for builder in builders_list %}
-{% set config_baseurl = salt['pillar.get']('build-infra:build-envs:' + env + ':builders-list:' + builder + ':config:repository:baseurl', 'https://github.com/QubesOS/qubes-') %}
+{% set config_baseurl = salt['pillar.get']('build-infra:build-envs:' + env + ':builders-list:' + builder + ':config:repository:baseurl', 'ssh://github.com/QubesOS/qubes-') %}
```

Comment:
No. I don't want any more secrets available in the build vm. And definitely don't want it to have write access to some repositories (which with github permissions model would require creating yet another separate github account).